wake-word-voice-assistant/m5stack-atom-echo.yaml

```diff
@@ -1 +1,352 @@
-<<: !include ../voice-assistant/m5stack-atom-echo.yaml
```

Comment:
I suggest updating the existing file instead of changing this import flow.
This will break existing implementations that are still using the "old" file for update checks.

Reply:
If the concern is the breaking change, the minimum ESPHome version enforcement should be enough.